### PR TITLE
[RFR] Added “Universal” entry

### PR DESCRIPTION
**Pull request by rbrtsmith**
Isomorphic also commonly known as universal, this list should contain both IMO-

### CODE REVIEW

**Comment by KittyGiraudel** [comment]
Could you make the file a `.md` please? :)

**Comment by rbrtsmith** [comment]
Done :+1:

File: glossary/UNIVERSAL.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Universal
+
+A universal (isomorphic) application is one whose code (in this case, JavaScript) can run both in the server and the client.
```
Comment (KittyGiraudel) [inline glossary/UNIVERSAL.md]: Maybe we could add a link to isomorphic entry here? What do you think?

Comment (rbrtsmith) [inline glossary/UNIVERSAL.md]: I think that's a good idea.  Do you want me to fix that?

On 6 December 2015 at 20:48, Hugo Giraudel notifications@github.com wrote:

> In glossary/UNIVERSAL.md
> https://github.com/HugoGiraudel/SJSJ/pull/46#discussion_r46774884:
> 
> > @@ -0,0 +1,7 @@
> > +# Universal
> > +
> > +A universal (isomorphic) application is one whose code (in this case, JavaScript) can run both in the server and the client.
> 
> Maybe we could add a link to isomorphic entry here? What do you think?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/HugoGiraudel/SJSJ/pull/46/files#r46774884.

## 

Kind regards,

_\- Robert Smith_
_Web Developer_

---

_W_: http://rbrtsmith.com/
_T_: +46(0)73 687 5773
_E_: rbrtsmith@googlemail.com

Comment (KittyGiraudel) [inline glossary/UNIVERSAL.md]: If you don’t mind. Else I’ll do it.

Comment (rbrtsmith) [inline glossary/UNIVERSAL.md]: Ok, It should be done now.

On 6 December 2015 at 21:06, Hugo Giraudel notifications@github.com wrote:

> In glossary/UNIVERSAL.md
> https://github.com/HugoGiraudel/SJSJ/pull/46#discussion_r46775117:
> 
> > @@ -0,0 +1,7 @@
> > +# Universal
> > +
> > +A universal (isomorphic) application is one whose code (in this case, JavaScript) can run both in the server and the client.
> 
> If you don’t mind. Else I’ll do it.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/HugoGiraudel/SJSJ/pull/46/files#r46775117.

## 

Kind regards,

_\- Robert Smith_
_Web Developer_

---

_W_: http://rbrtsmith.com/
_T_: +46(0)73 687 5773
_E_: rbrtsmith@googlemail.com

**Pull request merged**